app/services/breadcrumbs_options.rb

```diff
@@ -30,11 +30,15 @@ def index_item
     end
 
     {
-      caption: _(resource_name.pluralize.humanize),
-      url: resource_path(class_name)
+      caption: get_caption(resource_name),
+      url: resource_path(class_name) || resource_path(resource_name)
     }
   end
 
+  def get_caption(resource_name)
+    (resource_name == 'operatingsystem') ? _('Operating Systems') : _(resource_name.pluralize.humanize.titleize)
```

Comment:
is `titleize` needed?

Reply:
I think the names should be titleized
They are also titleizes in the navigation menu

Reply:
so the humanize is redundant?

Reply:
both needed
`'compute_profiles'.humanize`  - Compute profiles
`'compute_profiles'.humanize.titleize` -  Compute Profiles